лекция4

Podhaluzin opened the pull request.

Podhaluzin committed:
,

Podhaluzin committed:
.

Podhaluzin committed:
Delete DZ01.py

Podhaluzin committed:
Delete DZ02.py

Podhaluzin committed:
Delete DZ03.py

Podhaluzin committed:
Delete DZ04.py

Podhaluzin committed:
Delete DZ05.py

Podhaluzin committed:
Delete DZ06.py

Podhaluzin committed:
Delete README.md

pantonov-gb commented on lesson4-01.py:
```diff
@@ -0,0 +1,10 @@
+        print(f'заработная плата сотрудника  {res}')
+    except ValueError:
+        return print('Not a number')
+sal()
```
необходимо реализовать передачу параметров из командной строки через sys.argv, а не запрашивать ввод у пользователя.

Podhaluzin replied:
Понял. Пересдам, если успею до завтра

pantonov-gb commented on lesson4-02.py:
```diff
@@ -0,0 +1,4 @@
+my_list = [111, 24, 199, 12, 125, 103, 28, 10]
+my_new_list = [el for num, el in enumerate(my_list) if my_list[num - 1] < my_list[num]]
```
В первую итерацию какие элементы сравниваются между собой?

Podhaluzin replied:
Ага,я понял. Я к отрицательному индексу обращаюсь. И поэтому перерасчет. Правильно? Переделаю

pantonov-gb commented on lesson4-06(a).py:
```diff
@@ -0,0 +1,4 @@
+from itertools import count
+
+for el in count(int(input('Введите стартовое число '))):
+    print(el)
```
Бесконечный цикл

Podhaluzin replied:
Допишу код, который будет делать выход

pantonov-gb commented on lesson4-06(b).py:
```diff
@@ -0,0 +1,5 @@
+
+my_list = [True, 'ABC', 123, None]
+for el in cycle(my_list):
+    print(el)
```
Бесконечный цикл. По условию нужно предусмотреть выход

Podhaluzin replied:
Допишу код, который будет делать выход

pantonov-gb reviewed (commented):
Нужно продолжить работу, исправить недочеты

Podhaluzin committed:
Добавил правильное решение 1 задания

Реализовал передачу данных через командную строку (sys,argv)

Podhaluzin committed:
Переделал одну строку во 2 задании

Вроде, я так понял, нужно поменять [num-1] местами в списках))

Podhaluzin committed:
Переделал 6(a) задание

Я не организовал выход из бесконечного цикла, а просто сделал расчет до 70. так вроде даже проще.

Podhaluzin committed:
Переделал задание 6(b)

Получилось как-то так.

pantonov-gb reviewed (approved):
Ok
